docs/security-monitoring.md

```diff
@@ -23,11 +23,16 @@ Organizations using the agency add-on have the option to disable security monito
 
 ### Configuring Notifications
 Every user receives security notifications by email for all packages they have access to by default.
-Users can unsubscribe either from individual packages, or from all security notifications, if they not wish to receive any email notifications.
+Users can unsubscribe either from individual packages or from all security notifications if they do not wish to receive any email notifications.
 
-In addition to registered users receiving security notifications by email, you can configure notifications to be sent
-to other email addresses, e.g. a security mailinglist, or to a Slack channel.
-Create a notification channel for the desired destination and configure which notifications it should receive in the organization's security settings.
+Notification channels allow you to receive security notifications via other means than email to user accounts. The following types of notification channels are available:
+- **Email**: Sends notifications to a list of email addresses
+- **Slack** Webhook: Receives notifications in your configured Slack channel
```

Comment:
why does email send notifications but slack webhook receives notifications? That's a bit confusing, shouldn't it be "Sends notifications to your configured Slack channel"?